.gitignore

```diff
@@ -1,4 +1,5 @@
 .nyc_output
+.nyc_cache
```

Comment:
Could this be put in the home directory instead? It's annoying having to .gitignore this in every repo using nyc. ESLint does this by default: https://github.com/eslint/eslint/blob/master/docs/user-guide/command-line-interface.md#--cache-location